File: 250107/약수가 세 개인 수/numbers-with-three-divisors.py
start, end = map(int, input().split())
cnt=0
# Write your code here!
for i in range(start, end+1):
    tmpcnt = 0
    for j in range(1, i+1):
        if i%j==0:
            tmpcnt+=1
    if tmpcnt == 3:
        cnt+=1
print(cnt)

File: 250811/약수가 세 개인 수/numbers-with-three-divisors.js
const fs = require('fs');
let input = fs.readFileSync(0).toString().trim().split(" ");
let start = Number(input[0]);
let end = Number(input[1]);
let result = 0


function check(number){
    let cnt = 0;
    for(let i=1;i<=number;i++){
        if (number%i===0){
            cnt+=1;
        }
    }
    if (cnt === 3){
        return true;
    }
}

for (let j = start; j<=end; j++){
    if (check(j)){
        result+=1;
    }
}

console.log(result);